test/models/audit_trail_test.rb

```diff
@@ -31,24 +35,34 @@ class AuditTrailTest < ActiveSupport::TestCase
                    @track.date_of_change
     end
 
+
     it 'should conveniently render changed fields' do
       assert_equal @track.changed_fields,
-                   ['Name', 'Primary phone', 'Identifier']
+                   ["Name", "Primary phone", "Appointment date", "Initial call date", "Identifier"]
     end
 
     it 'should conveniently render what they were before' do
       assert_equal @track.changed_from,
-                   ['Susie Everyteen', '1112223333', 'D2-3333']
+                   ["Susie Everyteen",
+                     "1112223333",
+                     (Time.now + 5.days).strftime('%m/%d/%Y'),
```

Comment:
Swap these Time.now to Time.zone.nows? That way they won't surprise choke on time zone weirdness for no reason between the hours of, like, midnight and 4am or whatever. May also be able to use `display_date` in tests.